console/src/components/validateLogin.js

```diff
@@ -0,0 +1,86 @@
+import { loadAccessKeyState, clearState } from './AppState';
```

Comment:
Shouldn't this file be placed under a folder? Like src/components/Common/validateLogin.js ?